README.md

```
@@ -286,14 +294,14 @@ To connect using Websockets you can use the same DNS names as above but change `
 # JSON-RPC API
 The EVM Gateway implements APIs according to the Ethereum specification: https://ethereum.org/en/developers/docs/apis/json-rpc/#json-rpc-methods.
 
-**Additional APIs**
+## Additional APIs
 - Tracing APIs allows you to fetch execution traces
     * `debug_traceTransaction`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)